### PR TITLE
Deletion Phase 0: Test callability of a draft function

### PR DESCRIPTION
**Pull request by acgrobman**
This is needed because of cross origin resource sharing it has to run on Google not locally.

### CODE REVIEW

**Review by akampen — APPROVED** [review]

File: functions/.eslintrc.json [hunk 1]
```diff
@@ -99,7 +99,9 @@
           "Subcollection",
           "breakpoint",
           "Breakpoint",
-          "purpleair"
+          "purpleair",
+          "micrograms",
+          "Recurse"
```
Comment (CarsonHerness) [inline functions/.eslintrc.json:104]: Remove micrograms and Recurse as they're no longer used, add `uid`

File: functions/src/index.ts [hunk 2]
```diff
@@ -28,3 +30,34 @@ exports.generateReadingsCsv = functions
...
+      .get();
+    return userDocument.data()?.isAdmin ?? false;
+  }
+
+  return false;
+}
+
+exports.testCallable = functions.https.onCall(async (context: EventContext) => {
+  if (!(await isAdmin(context))) {
+    console.log('Not an admin');
```
Comment (CarsonHerness) [inline functions/src/index.ts:55]: just add an eslint-disable line

File: functions/src/index.ts [hunk 3]
```diff
@@ -28,3 +30,34 @@ exports.generateReadingsCsv = functions
 exports.generateAverageReadingsCsv = functions.pubsub
   .topic('generate-average-readings-csv')
   .onPublish(generateAverageReadingsCsv);
+
+/* exports.deleteOldReadings = functions
```
Comment (CarsonHerness) [inline functions/src/index.ts:34]: plz remove the commented code for the submitted code

File: functions/src/index.ts [hunk 4]
```diff
@@ -2,9 +2,11 @@ import {
   generateReadingsCsv,
   generateAverageReadingsCsv,
 } from './download-readings';
-import {functions} from './admin';
+import {firestore, functions} from './admin';
 import {calculateAqi} from './aqi-calculation/calculate-aqi';
 import {purpleAirToFirestore} from './aqi-calculation/purple-air-response';
+//import deleteOldReadings from './delete-old-readings';
```
Comment (CarsonHerness) [inline functions/src/index.ts:8]: plz remove the commented out code for submitted code

**Review by CarsonHerness — APPROVED** [review]

File: functions/package.json [hunk 5]
```diff
@@ -26,10 +26,10 @@
     "firebase-functions": "^3.13.2"
   },
   "devDependencies": {
-    "typescript": "^4.2.3",
+    "@types/node": "^14.14.37",
```
Comment (CarsonHerness) [inline functions/package.json:29]: nit: remove these changes

Comment (acgrobman) [reply functions/package.json:29]: This is more alphabetical, so i'mma keep it, also I didn't do this so I'm not sure how this happened.

**Pull request merged**